lib/validate.js

```diff
@@ -77,14 +84,20 @@ module.exports = {
     const entries = {};
 
     const functions = this.serverless.service.getAllFunctions();
+
     if (this.options.function) {
       const serverlessFunction = this.serverless.service.getFunction(this.options.function);
-      const entry = getEntryForFunction.call(this, this.options.function, serverlessFunction);
-      _.merge(entries, entry);
-    } else {
-      _.forEach(functions, (func, index) => {
-        const entry = getEntryForFunction.call(this, functions[index], this.serverless.service.getFunction(func));
+      if (this.hasNodeRuntime(serverlessFunction)) {
```

Comment:
@johnf Did you check if this works with `serverless invoke local` in case you invoke a non-node function?
The entries object will end up empty in this case (because the single function is skipped - which is ok), but what will webpack do in this case?

Reply:
@HyperBrain 

I've done a bunch of testing of different scenarios. Amended the code to skip running webpack in the right places.

One caveat though, in a mixed project you have to package individually. This is because if you just do an ```sls deploy``` the serverless-webpack code takes over and only packages up the js files. It will ignore the rest of the project.

I've added a note to the README. Feel free to edit or move it somewhere else in there.